browser-app/wdio.base.conf.js

```diff
@@ -0,0 +1,288 @@
+            return require('./src-gen/backend/server')(port, host, argv).then(created => {
+                this.execArgv = [wdioRunnerScript, cliPortKey, created.address().port,
+                    '--theia-root-dir', rootDir
+                ];
+                this.server = created;
+            });
+        },
+        // Gets executed after all workers got shut down and the process is about to exit. It is not
+        // possible to defer the end of the process using a promise.
+        onComplete: function (exitCode) {
+            if (this.server) {
+                this.server.close();
+            }
+        },
+        //
+        // Gets executed just before initialising the webdriver session and test framework. It allows you
+        // to manipulate configurations depending on the capability or spec.
+        // beforeSession: function (config, capabilities, specs) {
+        // },
+        //
+        // Gets executed before test execution begins. At this point you can access all global
+        // variables, such as `browser`. It is the perfect place to define custom commands.
+        // before: function (capabilities, specs) {
+        // },
+        //
+        // Hook that gets executed before the suite starts
+        // beforeSuite: function (suite) {
+        // },
+        //
+        // Hook that gets executed _before_ a hook within the suite starts (e.g. runs before calling
+        // beforeEach in Mocha)
+        // beforeHook: function () {
+        // },
+        //
+        // Hook that gets executed _after_ a hook within the suite starts (e.g. runs after calling
+        // afterEach in Mocha)
+        // afterHook: function () {
+        // },
+        //
+        // Function to be executed before a test (in Mocha/Jasmine) or a step (in Cucumber) starts.
+        // beforeTest: function (test) {
+        // },
+        //
+        // Runs before a WebdriverIO command gets executed.
+        // beforeCommand: function (commandName, args) {
+        // },
+        //
+        // Runs after a WebdriverIO command gets executed
+        // afterCommand: function (commandName, args, result, error) {
+        // },
+        //
+        // Function to be executed after a test (in Mocha/Jasmine) or a step (in Cucumber) starts.
+        // afterTest: function (test) {
+        // },
+        //
+        // Hook that gets executed after the suite has ended
+        afterSuite: function (suite) {
+            require("webdriverio");
+            var fs = require("fs");
+            let result;
+            try {
+                result = browser.execute("return window.__coverage__;");
+            } catch (error) {
+                console.error(`Error retrieving the coverage: ${error}`);
+                return;
+            }
+            try {
+                if (!fs.existsSync('coverage')) {
+                    fs.mkdirSync('coverage');
```

Comment:
I remember now: a "coverage" dir is created after the wdio tests run, and looking at the content we should be able to see if in there if it seems to run